pkgs/by-name/be/beidconnect/package.nix

```
@@ -43,13 +31,20 @@ stdenv.mkDerivation (finalAttrs: {
 
   postInstall = ''
     install -d \
-      $out/native-messaging-hosts/chromium/ \
-      $out/native-messaging-hosts/mozilla/ \
+      $out/etc/chromium/native-messaging-hosts \
+      $out/etc/opt/chrome/native-messaging-hosts/ \
+      $out/etc/opt/edge/native-messaging-hosts/ \
+      $out/usr/lib/mozilla/native-messaging-hosts \
+      $out/usr/lib64/mozilla/native-messaging-hosts
```

```suggestion
      $out/lib/mozilla/native-messaging-hosts
```

We don’t use `usr` or `lib64`. Other than that this looks good to me, thank you for packaging this!